site/utils/apiUtils/disruptionApprovalEmailer.ts

```diff
@@ -0,0 +1,139 @@
+                                               ></path></svg
+                                             ><span style="padding-left: 5px;"
+                                               >GOV.UK</span
+                                             ></span
+                                           ></a
+                                         >
+                                       </div>
+                                     </div>
+                                   </header>
+                                  <div style="font-family: Arial, sans-serif; padding-left: 30px">
+                       <h1>Approval Required - Create Transport Disruptions Data service</h1>
+                       <p>A disruption has been created that requires your approval before it can be published.</p>
+                       <p>Disruption Summary: ${disruptionSummary}</p>
+                       <p>Disruption Description: ${disruptionDescription}</p>
+                       <p>Submitted by: ${disruptionCreator}</p>
+                       <a href=${disruptionLink}>Link to disruption</a>
+                    </div>
+                                 </div>
+                               </div>
+                             </body>
+                           </html>`;
+
+export const createDisruptionApprovalEmail = (
+    disruptionSummary: string,
+    disruptionDescription: string,
+    disruptionCreator: string,
+    disruptionId: string,
+    orgAdminEmails: string[],
+) => {
+    const disruptionLink = isSandbox(STAGE)
+        ? `http://www.localhost:3000/disruption-detail/${disruptionId}`
+        : `${getDomain(STAGE)}`;
+
+    return new SendEmailCommand({
+        Destination: {
+            ToAddresses: orgAdminEmails,
+        },
+        Message: {
+            Body: {
+                Html: {
+                    Charset: "UTF-8",
+                    Data: approvalEmailBody(
+                        disruptionSummary,
+                        disruptionDescription,
+                        disruptionCreator,
+                        disruptionLink,
+                    ),
+                },
+            },
+            Subject: {
+                Charset: "UTF-8",
+                Data: "Action required for the Create Disruption Data service",
+            },
+        },
+        Source: `no-reply@${getDomain(STAGE)}`,
+    });
+};
+
+export const getOrgAdminEmailsForStaffUserOrg = (orgAdminsForAllOrgs: UserType[], staffOrgId: string) => {
+    return orgAdminsForAllOrgs
```

Comment:
find or filter function can be used here to obtain the org admins based on staff org id if the data returned from `getAllUsersInGroup` is parsed by a schema. All data from external sources are parsed using zod.

Reply:
refactored